ts/utils/convertDateToWordDistance.ts

```diff
@@ -20,7 +22,12 @@ export function convertDateToWordDistance(
     return lastDayLabel;
   } // 1 day < distance < 365 days, current year
   else if (distance > 1 && distance < 365) {
-    return format(date, "DD/MM");
+    const locale = I18n.currentLocale();
```

Comment:
The problem with this solution is that we have to change the code each time we add a new language. I think you can use the utility function exported here https://github.com/teamdigitale/io-app/blob/master/ts/utils/dates.ts#L26.

Reply:
Please check the comment above.

Reply:
Thank you for feedback. I'm looking for it but it seems date-fns doesn't provide a function which makes the date format depends on the passed locale. I could implement the function in date.ts and pass it to convertDateToWordDistance.ts . What do you think?

Reply:
Another possible solution https://github.com/fnando/i18n-js#date-formatting